US-Trustee-Program/Bankruptcy-Oversight-Support-Systems
# VER-11 Add content security policy

# Problem

See issue VER-11 for details

# Solution

Define initial content-security-policy

# Testing/Validation

- Verify in branch deployment with included header updates in nginx.conf
- Verify finding resolved with Veracode DAST scan

# Other Changes
